### DISCUSSION TITLE
Already installed and configured language servers and null-ls not detected

### DISCUSSION THREAD

**Discussion by bokonist** [mason-org/mason.nvim#32] [Q&A] [answered]
Hey, thank you for making this. Currently it does not detect any already running and attached language servers. This is what :LspInfo outputs:

![image](https://user-images.githubusercontent.com/23412356/178063712-249919ce-9d77-4b35-9145-4c12c7ba2259.png)

Here's what :Mason shows:

![image](https://user-images.githubusercontent.com/23412356/178063474-c5ef7460-a33f-4474-ac67-2890d375fc36.png)

**Comment by williamboman** [comment, marked as answer]
> Hey, thank you for making this. Currently it does not detect any already running and attached language servers. 

Hello! Hmm I'm not sure I'm following, but why would this plugin display tooling that are installed externally? If you install the servers via mason, they'll show up as installed there! The UI window is not meant to be an interface for things installed outside of mason.nvim :)

**Comment by bokonist** [comment]
@williamboman Makes sense, I thought it might be compatible with lspinstaller

**Reply by williamboman** [reply]
Ah ok. It depends on where you look - I'd argue it is fully compatible via the `mason-lspconfig` extension (docs needed). It will however not use the same installation path as nvim-lsp-installer because the file hierarchy is completely different, so it'll start off from a clean slate. 

Remember to clean up the nvim-lsp-installer installations if you decide to use mason.nvim so that they dont occupy disk space (:LspUninstallAll` is one way to so it, alternatively manually removing the entire installation directory)

**Reply by bokonist** [reply]
thanks, Will! I'll try and make it work by resinstalling the servers with mason
